src/popup/router/pages/Index.vue

```diff
@@ -74,6 +77,7 @@ export default {
   data: () => ({
     termsAgreed: false,
     IS_WEB: process.env.PLATFORM === 'web',
+    IS_MOBILE: process.env.PLATFORM === 'cordova',
```

Comment:
```suggestion
    IS_MOBILE: window.IS_MOBILE_DEVICE,
```
Do cordova have a special user agent not containing `Mobi`? Or if it is only for cordova then I would rename the variable to `IS_CORDOVA`.